flowauth/frontend/src/ServerAdminDetails.jsx

```diff
@@ -167,24 +168,27 @@ class ServerAdminDetails extends React.Component {
     const { item_id } = this.props;
     if (item_id !== -1) {
       try {
-        const capAwait = getCapabilities(item_id);
+        const capAwait = getScopes(item_id);
         const limitsAwait = getTimeLimits(item_id);
-        const scopeGraph = scopesGraph(await capAwait);
+        const rights = await capAwait.then(
+            resp => resp.reduce((p_x,x) => ({[x.name]:x.enabled, ...p_x}), {})
+          );
+        console.log("Rights", rights)
```

Comment:
Same here: this looks like a tmp log; I'd either reduce it or remove it entirely.